assets/app/views/settings.html

```diff
@@ -1,119 +1,233 @@
 <div class="content">
   <project-page>
-    <div>
-      <div class="page-header page-header-bleed-right">
-        <delete-link
-          class="pull-right fa-lg"
-          resource-type="project"
-          resource-name="{{project.metadata.name}}"
-          project-name="{{project.metadata.name}}"
-          display-name="{{(project | displayName)}}"
-          alerts="alerts"
-          button-only>
-        </delete-link>
-        <h1>Project Settings</h1>
+    <div class="breadcrumb"></div>
+    <alerts alerts="alerts"></alerts>
+
+      <h1>
+        Project Settings
+        <div class="pull-right dropdown">
+          <a href=""
+            class="dropdown-toggle resource-actions-dropdown"
+            data-toggle="dropdown">
+            <i class="fa fa-ellipsis-v"></i>
+            <span class="sr-only">Actions</span>
+          </a>
+          <ul class="dropdown-menu actions action-button">
+            <li>
+              <!--
+                TODO: gross, this is hacky, but gets the view state correct...
+              -->
+              <a
+                ng-if="!show.editing"
+                href=""
+                role="button"
+                ng-click="toggle()">
+                <i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit
+              </a>
+              <button
+                ng-if="show.editing"
+                type="button"
+                class="btn btn-link"
+                DISABLED>
```

Comment:
Right, I'm preferring that disabled wouldn't even let you click, rather than a click that does nothing.

Buttons can be used inside menus, but unfortunately we lose the hover effect:

![screen shot 2016-01-18 at 1 25 03 pm](https://cloud.githubusercontent.com/assets/280512/12399298/1f707172-bde7-11e5-828b-12de504102d3.png)

"Edit" is a link, will give the blue outline.  "Edit 2" is a button... will not outline.  

Silly that this isn't nicely handled by bootstrap menu. :unamused:  I'll tinker a bit.